crates/runtime/src/instance.rs

```diff
@@ -73,6 +75,9 @@ pub(crate) struct Instance {
     /// interrupted.
     pub(crate) interrupts: Arc<VMInterrupts>,
 
+    /// A canonical `ref.null func` value for this instance.
+    pub(crate) null_func_ref: Box<VMCallerCheckedAnyfunc>,
```

Comment:
I saw somewhere else that this was store-local, but it's also here per-instance? Could this perhaps be `*mut` and stored in the `Store` instad?

Reply:
FWIW, we don't use pointer equality for `VMCallerCheckedAnyfunc`s, so this would just be a memory optimization that saves two words per instance.

Reply:
Ah yeah that's all I'm imagining, fewer small allocations here-and-there. It also sort of helps the case of "I precompiled my wasm" since we'd presumably statically allocate this anyfunc instead of trying to dynamically allocate it.